### PR TITLE
mageia 6 Image update 

### PR DESCRIPTION
**Pull request by juanluisbaptiste**
Hi,

There's an updated mageia 6 Image that fixes juanluisbaptiste/docker-brew-mageia#7.

Thanks

### CODE REVIEW

File: library/mageia [hunk 1]
```diff
@@ -2,11 +2,11 @@ Maintainers: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
...
-GitCommit: 6230a3fc5dd30e5419a391f4613eb3ca5d81f46a
+GitCommit: e690ea85472945459e1401c4dc6b7568f11d1e2c
 GitFetch: refs/heads/dist
 Directory: 6
 
 Tags: 5,5.1
-GitCommit: 6230a3fc5dd30e5419a391f4613eb3ca5d81f46a
+GitCommit: e690ea85472945459e1401c4dc6b7568f11d1e2c
 GitFetch: refs/heads/dist
 Directory: 5
```
Comment (tianon) [inline library/mageia]: The `Dockerfile` in this directory is referencing `rootfs.tar.xz`, but the directory contains `rootfs.tar.gz` -- was that an intentional mixup? :innocent:

**Comment by juanluisbaptiste** [comment]
Ahhhhh dammit, I didn't notice that... fixing...

**Comment by juanluisbaptiste** [comment]
Fixed.

File: library/mageia (hunk 1 above)
Comment (juanluisbaptiste) [reply library/mageia]: Fixed.

**Comment by tianon** [comment]
Diff: https://gist.github.com/docker-library-bot/f341ae5c794e34d7f7afc44c0474fa72

LGTM

Build test of #4290; 07189ca6; `amd64` (`mageia`):

```console

$ bashbrew build mageia:6
Building bashbrew/cache:1cb3763c08f4f0f52745c316c2fb35e617d7aff3ab93452a8c7e00dff9c202e6 (mageia:6)
Tagging mageia:6
Tagging mageia:latest

$ test/run.sh mageia:6
testing mageia:6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build mageia:5
Building bashbrew/cache:4357566bbcb74e84d5e348115af2ce6d5cf66995cda1d6de44fb0cdb266197d0 (mageia:5)
Tagging mageia:5
Tagging mageia:5.1

$ test/run.sh mageia:5
testing mageia:5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**